### PR TITLE
Miscellaneous small stuff as I got it running

### PR DESCRIPTION
**Pull request by mvidner**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,6 +42,11 @@ Installation
   4. Install [Pennyworth](https://github.com/SUSE/pennyworth#installation) but instead
      of cloning the repository, install the
      [pennyworth-tool gem](https://rubygems.org/gems/pennyworth-tool/).
+     An older version is necessary now because of an API change
+     in `Pennyworth::VM#run_command`.
+
+        $ gem install pennyworth-tool --no-format-executable --version "= 0.1.0"
```
Comment (imobach) [inline README.md]: I use 0.2.0 and it works. I'll recheck anyway.

File: build_iso/sles12-sp1.rb [hunk 2]
```diff
@@ -35,7 +43,7 @@
 
 Dir.chdir(iso_dir) do
   puts "\n**** Downloading source ISO image ****"
-  system "wget --no-clobber #{iso_url}"
+  system "wget --progress=dot:giga --no-clobber #{iso_url}"
```
Comment (imobach) [inline build_iso/sles12-sp1.rb]: Nice :)

File: build_iso/sles12.rb [hunk 3]
```diff
@@ -18,9 +18,17 @@
 require "rubygems"
 require "fileutils"
 
+def iso_repo
```
Comment (imobach) [inline build_iso/sles12.rb]: That reminds me that we need to do some refactor to avoid repeating code.

**Comment by imobach** [comment]
LGTM

File: README.md (hunk 1 above)
Comment (mvidner) [inline README.md]: With 0.2.0 I was getting
```
Failures:

  1) SLES 12 TFTP server  checks, tftp is running correctly
     Failure/Error: run_test_script("tftp.sh")
     NoMethodError:
       undefined method `split' for #<Pennyworth::VM::CommandResult:0x00000002b4c808>
     # ./helper/spec_helper.rb:30:in `run_test_script'
     # ./spec/tftp.rb:5:in `block (2 levels) in <top (required)>'

  2) SLES 12 TFTP server  checks, if host is in /etc/hosts
     Failure/Error: run_test_script("host.sh")
     NoMethodError:
       undefined method `split' for #<Pennyworth::VM::CommandResult:0x00000002b449a0>
     # ./helper/spec_helper.rb:30:in `run_test_script'
     # ./spec/tftp.rb:10:in `block (2 levels) in <top (required)>'

  3) SLES 12 TFTP server  checks, whether unsupported YaST modules are reported
     Failure/Error: run_test_script("unsupported_modules.sh")
     NoMethodError:
       undefined method `split' for #<Pennyworth::VM::CommandResult:0x00000002b3cc00>
     # ./helper/spec_helper.rb:30:in `run_test_script'
     # ./spec/tftp.rb:15:in `block (2 levels) in <top (required)>'

```

and indeed, [Command Result](https://github.com/SUSE/pennyworth/blob/92f43c1553c6d75fbd840d231e5cdb674e593f5a/lib/pennyworth/vm.rb#L24) does not have a split method. (0.1.0 returned a Cheetah result, which is just the captured output)

File: build_iso/sles12.rb (hunk 3 above)
Comment (mvidner) [inline build_iso/sles12.rb]: Yes, especially the main Rakefile is quite low level, and does not help you if you still find your way around all the unfamiliar tools and all the images and VMs that are being handled.

File: README.md (hunk 1 above)
Comment (imobach) [inline README.md]: Oh, I see, I've checked and it looks like I have both versions installed. Then I think we should add the version to the Gemfile.

File: build_iso/sles12.rb (hunk 3 above)
Comment (imobach) [inline build_iso/sles12.rb]: I guess that now that we want to support VirtualBox, we'll do some refactoring.

**Pull request merged**